projects/plugins/crm/includes/jpcrm-templating-placeholders.php

```diff
@@ -1755,10 +1755,31 @@ public function replace_placeholders(
 					
 					}
 
+					// If this is a Quote date key and is not set (Quote accepted or last viewed), let's print out a message saying the quote isn't accepted or viewed.
+					$possible_empty_quote_keys = array( 'quote-accepted', 'quote-accepted_date_str', 'quote-accepted_datetime_str', 'quote-lastviewed', 'quote-lastviewed_datetime_str', 'quote-lastviewed_date_str' );
+					if ( in_array( $key, $possible_empty_quote_keys, true ) ) {
+						// 82799 is 23hrs, 59mins, 59secs on Jan 1, 1970 (so a blank timestamp, allowing for systems to update the time within that day for any reason)
+						if ( empty( $replace_with ) || strtotime( $replace_with ) <= 82799 ) {
```

Comment:
Why not `=== 0`?

Reply:
In my case, the timestamp for 1 Jan 1970 was 3600 (+1hr for the exact time), so I figured that to cover any timezone offsets but still be 1st Jan then that figure should work.

Reply:
> In my case, the timestamp for 1 Jan 1970 was 3600

To confirm, it stored `3600` in the database when it was not accepted? It seems `0` is the value other queries use elsewhere, so if you're seeing anything else, it sounds like a different bug.

Reply:
Looking at the code, it should always set it to `0` (not the Unix timestamp + timezone offset equivalent of 1 Jan 1970) if not accepted. I'd stick to `0` for any comparisons.

Reply:
(in my ideal world, we'd never use `0` for timestamps (or `-1`), but legacy code...)

Reply:
> To confirm, it stored 3600 in the database when it was not accepted? 
That was the result after the str_to_time conversion. I'm switching to using `jpcrm_date_str_to_uts` though now.